src/kernel_config_manager.sh

```diff
@@ -562,14 +560,15 @@ function remove_config()
   basic_config_validations "$target" "$force" 'Remove' "$msg"
 
   cd "$dot_configs_dir" || exit_msg 'It was not possible to move to configs dir'
-  git rm "$configs_dir/$target" "$dot_configs_dir/$metadata_dir/$target" > /dev/null 2>&1
-  git commit -m "Removed $target config: $USER - $(date)" > /dev/null 2>&1
+  if ! rm "$configs_dir/$target" "$dot_configs_dir/$metadata_dir/$target"; then
+    exit_msg 'Could not remove config file'
+  fi
   cd "$original_path" || exit_msg 'It was not possible to move back from configs dir'
 
   say "The $target config file was removed from kw management"
 
   # Without config file, there's no reason to keep config directory
-  if [ ! "$(ls "$dot_configs_dir")" ]; then
+  if [ ! "$(ls "$dot_configs_dir"/"$configs_dir")" ]; then
```

Comment:
https://kworkflow.org/content/codingstyle.html#string-concatenation

Also, can we move this `ls` outside the `if` condition?